idea/testData/inspectionsLocal/removeRedundantBackticks/.inspection

```diff
@@ -0,0 +1 @@
+org.jetbrains.kotlin.idea.inspections.RemoveRedundantBackticksInspection
```

Comment:
Please add a test to verify that the quickfix is not offered when an actual keyword is in backticks.